ansible/playbooks/openadk.yaml

```
@@ -0,0 +1,48 @@
+- name: Set up source development environments for Autoware Universe
+  hosts: localhost
+  connection: local
+  pre_tasks:
+    - name: Verify OS
+      ansible.builtin.fail:
+        msg: Only Ubuntu 22.04 is supported for this branch. Please refer to https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/.
+      when: ansible_distribution == 'Ubuntu' and ansible_distribution_version != '22.04'
```

I think the following would be more robust:
```suggestion
      when: ansible_distribution != 'Ubuntu' or ansible_distribution_version != '22.04'
```

---

done (can be resolved)